bigtop-packages/src/common/airflow/airflow-webserver.service

```diff
@@ -0,0 +1,35 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+[Unit]
+Description=Airflow webserver daemon
+After=network.target postgresql.service mysql.service redis.service rabbitmq-server.service
+Wants=postgresql.service mysql.service redis.service rabbitmq-server.service
+
+[Service]
+EnvironmentFile=/etc/default/airflow
+User=airflow
+Group=airflow
+Type=simple
+ExecStart=/usr/lib/airflow/bin/airflow webserver --pid /run/airflow/webserver.pid
```

Comment:
Do we need pid file here? If we need, why don't we use [PIDFile=](https://www.freedesktop.org/software/systemd/man/latest/systemd.service.html#PIDFile=) configuration? I think we don't need PIDFile with 'Type=simple' as the systemd can handle the process without it.

Reply:
[Airflow's webserver manages internal gunicorn process](https://github.com/apache/airflow/blob/2.10.4/airflow/cli/commands/webserver_command.py#L448-L456), and for that purpose, it seems that the pid file path should be explicitly passed from the command line rather than using systemd's PIDFile feature.